Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN if [[ $optional_dependencies =~ "fasttext" ]]; then \
 		apt-get update && \
 		apt-get install -y --no-install-recommends \
 			build-essential && \
+		pip install --upgrade pip setuptools --no-cache-dir && \
```

Comment:
Would it make sense to install `wheel` here as well? In my understanding, it might speed up installation of some Python packages - they can be installed from prebuilt wheels instead of rebuilt. Not sure if that is relevant for the Annif dependencies though.

Reply:
Yes it would. When installing deps for unit test runs in GH Action, there are messages like
`Using legacy 'setup.py install' for xxx, since package 'wheel' is not installed.`
I was confused as I did not see these messages in Docker builds, but that was just because `wheel` actually was available as it got installed as a dependency (of TensorFlow?). 

(I'll install wheel also for unit test runs.)